.sam-dev-params

```diff
@@ -0,0 +1 @@
+stage=dev
```

Comment:
also, create new files for `stage` and `prod`
instead of value dev use the above `name` in respective files.

Reply:
created.